content/en/real_user_monitoring/guide/upload-javascript-source-maps.md

```diff
@@ -8,7 +8,7 @@ further_reading:
 - link: '/real_user_monitoring/error_tracking/explorer'
   tag: 'Documentation'
   text: 'Visualize your Error Tracking data in the Explorer'
-- link: 'https://github.com/DataDog/datadog-ci/tree/457d25821e838db9067dbe376d0f34fb1a197869/src/commands/sourcemaps'
+- link: 'https://github.com/DataDog/datadog-ci/tree/master/packages/datadog-ci/src/commands/sourcemaps'
```

Comment:
This link wasn't broken, but adding a permalink can take users to a potentially outdated version of the repo.